content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -0,0 +1,296 @@
+
+#### Generate an asset list 
+
+1. Select your new seed list file by enabling the check box. 
+2. Click **Generate Asset Lists**. This will open the **Generate Asset List files** dialog. 
+3. In the list of platforms, select **pc** by enabling the check box.  
+4. Click **Browse...** to open the File Explorer. 
+5. Enter a name for your asset list and save. In this example, we'll use the name `game.assetlist`. 
+5. Navigate to the **Asset Lists** tab to verify the assets in `game.assetlist`. The assets are listed under the **Asset List** panel. 
+
+#### Bundle your assets
+
+1. Select your `game.assetlist` in the** Asset List Files** panel under the **Asset Lists** tab.
+
+2. Click **Generate Bundle**. This opens the **Generate Bundles** dialog. 
+
+3. Click **Browse...** to open the File Explorer. 
+
+4. Enter a name for your package file and save. In this example, we'll use the name `game_bundle.pak`.
+
+
+### Create a bundle for engine assets
+
+Now, you'll create a bundle for your project's engine assets. 
+
+#### Generate an asset list from default seed lists
+
+1. Navigate to the Seeds tab. 
+
+2. Select **Default Seed Lists** at the bottom of the **Seed List files** panel. Make sure to deselect other seed list files. 
+
+2. Click **Generate Asset Lists**. This will open the **Generate Asset List files** dialog. 
+
+3. In the list of platforms, select **pc** by enabling the check box.  
+
+4. Click **Browse...** to open the File Explorer. 
+
+5. Enter a name for your asset list and save. In this example, we'll use the name `engine.assetlist`. 
+
+5. Navigate to the **Asset Lists** tab to verify the assets in `engine.assetlist`. The assets are listed under the **Asset List** panel. 
+
+#### Bundle your assets
+
+1. Select your `engine.assetlist` in the** Asset List Files** panel under the **Asset Lists** tab.
+
+2. Click **Generate Bundle**. This opens the **Generate Bundles** dialog. 
+
+3. Click **Browse...** to open the File Explorer. 
+
+4. Enter a name for your package file and save. In this example, we'll use the name `engine_bundle.pak`.
+
+
+You now have two bundled content: `game_bundle.pak` and `engine_bundle.pak`. 
+
+
+### Add your bundles to install layout
+
+Next, add `game_bundle.pak` and `engine_bundle.pak` to `install/bin/Windows/release/Default`. You can remove the `engine.pak` file that was created earlier. 
+
+
+## Run the game launcher
+
+Now that you've packaged your assets and built your project game release layout, you are ready to run your project's Game Launcher.
+
+1. Open a command line window and change to your project's release build install directory.
+
+    - For monolithic release builds:
+        ```cmd
+        cd C:/o3de-projects/MyProject/install/bin/Windows/release/Monolithic
+        ```
+
+    - For non-monolithic release builds:
+        ```cmd
+        cd C:/o3de-projects/MyProject/install/bin/Windows/release/Default
+        ```
```

Comment:
```suggestion
    - For non-monolithic release builds:
        ```cmd
        cd C:/o3de-projects/MyProject/install/bin/Windows/release/Default
        ```

    - For monolithic release builds:
        ```cmd
        cd C:/o3de-projects/MyProject/install/bin/Windows/release/Monolithic
        ```
```
I re-ordered these two step alternatives here. Per an earlier comment, keep the order of "non-monolithic"/"monolithic" consistent.